homeassistant/components/awattar/config_flow.py

```diff
@@ -0,0 +1,43 @@
+"""Config flow for aWATTar integration."""
+
+from __future__ import annotations
+
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_COUNTRY_CODE
+
+from .const import DOMAIN
+
+COUNTRY_CODES = ["AT", "DE"]
+
+
+class AwattarFlowHandler(ConfigFlow, domain=DOMAIN):
+    """Config flow for aWATTar integration."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle the initial step."""
+
+        await self.async_set_unique_id(DOMAIN)
```

Comment:
Setting the unique_id doesn't prevent adding additional config entries with the same unique_id.
Also, maybe make the unique id the country code instead, and abort the flow if the user tries to set up another entry for the same country